Src/library/base/elks/kernel/type.e

```diff
@@ -1,12 +1,13 @@
 ﻿note
 	description: "Representation of an Eiffel type."
+	remark: "At any given time, there is no more than one instance of TYPE representing a given Eiffel type."
```

Comment:
If TYPED_POINTER [INTEGER] can be assigned to TYPED_POINTER [ANY], there is a type safety violation at run-time that can cause memory access violations or other bad things. That's why my first reaction was that conformance rules for expanded types might need to be stricter than for reference types. Alternatively, it might be required that is_equal and copy have to be redefined and cannot use standard_is_equal and standard_copy.
As to backward compatibility, there could be a breaking change, but before it happens, a tool can be used to find equality comparisons using = for objects of type TYPE.